schwifty/bank_registry/manual_it.json

```diff
@@ -662,5 +662,469 @@
+  },
+  {
+     "bank_code":"08358",
+     "bic":"ICRAITRR9Y0",
+     "country_code":"IT",
+     "name":"BANCA DI PESCIA E CASCINA - CREDITO COOPERATIVO - SOCIETA' COOPERATIVA",
+     "primary":true,
+     "short_name":"BANCA DI PESCIA E CASCINA"
+  },
+  {
+     "bank_code":"05262",
+     "bic":"BPPUIT33XXX",
+     "country_code":"IT",
+     "name":"BANCA POPOLARE PUGLIESE - SOCIETA' COOPERATIVA PER AZIONI",
+     "primary":true,
+     "short_name":"BANCA POPOLARE PUGLIESE"
+  },
+  {
+     "bank_code":"08704",
+     "bic":"ICRAITRRJK0",
+     "country_code":"IT",
+     "name":"BANCA DI CREDITO COOPERATIVO DI OSTRA E MORRO D'ALBA - SOCIETA' COOPERATIVA",
+     "primary":true,
+     "short_name":"BCC DI OSTRA E MORRO D'ALBA"
+  },
+  {
+     "bank_code":"08434",
+     "bic":"ICRAITRRC20",
+     "country_code":"IT",
+     "name":"BANCA DI CREDITO COOPERATIVO ABRUZZESE - CAPPELLE SUL TAVO SOCIETA' COOPERATIVA",
+     "primary":true,
+     "short_name":"BANCA DI CREDITO COOPERATIVO ABRUZZESE"
+  },
+  {
+     "bank_code":"05156",
+     "bic":"BCPCIT2PXXX",
+     "country_code":"IT",
+     "name":"BANCA DI PIACENZA - SOCIETA' COOPERATIVA PER AZIONI",
+     "primary":true,
+     "short_name":"BANCA DI PIACENZA"
+  },
+  {
+     "bank_code":"08462",
+     "bic":"CCRTIT2TROC",
+     "country_code":"IT",
+     "name":"BANCA DI CREDITO COOPERATIVO DELLA ROMAGNA OCCIDENTALE - SOCIETA' COOPERATIVA",
+     "primary":true,
+     "short_name":"BANCA DI CREDITO COOPERATIVO DELLA ROMAGNA OCCIDENTALE"
+  },
+  {
+     "bank_code":"08340",
+     "bic":"CCRTIT2TPAD",
+     "country_code":"IT",
+     "name":"CASSA PADANA BANCA DI CREDITO COOPERATIVO SOCIETA' COOPERATIVA",
+     "primary":true,
+     "short_name":"CASSA PADANA BANCA"
+  },
+  {
+     "bank_code":"08329",
+     "bic":"ICRAITRR950",
+     "country_code":"IT",
+     "name":"BANCA DI CREDITO COOPERATIVO BRIANZA E LAGHI - SOCIETA' COOPERATIVA",
+     "primary":true,
+     "short_name":"BANCA DI CREDITO COOPERATIVO BRIANZA E LAGHI"
+  },
+  {
+     "bank_code":"03399",
+     "bic":"EXRRITM2XXX",
+     "country_code":"IT",
+     "name":"<p><b>Bank:</b> EXTRABANCA S.P.A.</p>",
+     "primary":true,
+     "short_name":""
+  },
+  {
+     "bank_code":"08000",
+     "bic":"ICRAITRRXXX",
+     "country_code":"IT",
+     "name":"Iccrea Banca S.p.A. - Istituto Centrale del Credito Cooperativo",
+     "primary":true,
+     "short_name":""
+  },
+  {
+     "bank_code":"08399",
+     "bic":"ICRAITRRB30",
+     "country_code":"IT",
+     "name":"BANCA DELLE TERRE VENETE CREDITO COOPERATIVO - SOCIETA' COOPERATIVA",
+     "primary":true,
+     "short_name":"ANCA DELLE TERRE VENETE"
+  },
+  {
+     "bank_code":"08904",
+     "bic":"CCRTIT2TPRE",
+     "country_code":"IT",
+     "name":"BANCA PREALPI SANBIAGIO CREDITO COOPERATIVO - SOCIETA' COOPERATIVA",
+     "primary":true,
+     "short_name":"BANCA PREALPI SANBIAGIO"
+  },
+  {
+     "bank_code":"06090",
+     "bic":"CASRIT22XXX",
```

Comment:
I am not sure we kept the XXX pattern I cant see it in other banks, I summon @dennisxtria master of Italian banking to confirm.

Reply:
we want to avoid having `XXX` suffix as it causes an issue with railsbank, making us use the fallback method, which is slower from what @petros-an has told us, so ideally we want to avoid it